rfcs/convergence/move-out-n-rename-makestyles.md

```diff
@@ -0,0 +1,92 @@
+# RFC: Rename and move out `makeStyles` to a separate repo
+
+@layershifter
+
+## Summary
+
+`makeStyles()` is CSS-in-JS engine that is used in Fluent UI React v9. At this moment it seems that `makeStyles()` is more than just a piece of Fluent UI and there is sense to move out it to a separate repository for better visibility in community.
+
+An example there can be [Braid Design System](https://seek-oss.github.io/braid-design-system/) that uses [Vanilla Extract](https://github.com/seek-oss/vanilla-extract): design system and CSS-in-JS live in separate repositories.
+
+## Problem statement
+
+### No community engagement
+
+In Fluent UI repository we have already 4 projects, `makeStyles` stays invisible behind them 🙄 This results in inability to get community feedback and consumers outside Fluent UI.
+
+In the same time, community might have interest in `makeStyles` as it solves its goals efficiently. Taking into account that:
+
+- `makeStyles` is based on similar ideas as [Stylex](https://www.youtube.com/watch?v=9JZHodNR184&t=229s) from Facebook
+- we use module evaluation for [Linaria](https://github.com/callstack/linaria)
+
+It is a good opportunity to improve collaboration 🚀
+
+### `makeStyles` is coupled to Fluent UI
+
+Even if there is interest to use `makeStyles` separately from Fluent UI, it's currently impossible as it's coupled with Fluent UI React v9. There is a proposal how to decouple `makeStyles` (see RFC [microsoft/fluentui#20651](https://github.com/microsoft/fluentui/pull/20651)).
+
+But until it stays as a folder in Fluent UI repo there is no clarity about it for third parties.
+
+### Lack of docs
+
+Compared to other CSS-in-JS `makeStyles` does not have enough docs to explain APIs and design. While every other library has them 🙄 Currently there is no right place for them.
+
+### "Bad" naming
+
+It's something that was not considered initially, but `makeStyles` is also a function in [MUI](https://mui.com/styles/basics/) and any search queries will point to it.
+
+### Opportunities for partners
+
+Without strict dependency on Fluent UI v9 we can potentially involve more partners to be consumers of `makeStyles` as they will be able to try new styling engine even without consuming v9 components.
```

Comment:
```suggestion
Without a strict dependency on Fluent UI v9 we can potentially involve more partners to be consumers of `makeStyles` since they will be able to try the new styling engine even without consuming v9 components.

Partners using other component libraries can still see a performance improvement when using `makeStyles` with their components. This has been proven in the Teams chat list in T2.
```

Let's mention the benefits we've already seen just using makeStyles standalone